README.adoc

```diff
@@ -1,6 +1,6 @@
 // Do not edit this file (e.g. go instead to src/main/asciidoc)
```

Comment:
[Here](https://github.com/spring-cloud/spring-cloud-build/blob/master/docs/src/main/asciidoc/building-base.adoc), I'll get that one.